target/linux/ramips/image/mt7621.mk

```diff
@@ -518,6 +518,15 @@ define Device/youhua_wr1200js
 endef
 TARGET_DEVICES += youhua_wr1200js
 
+define Device/youku-yk2
```

Comment:
images are named as manufacturer_board so it should be Device/youku_yk-l2 here.

Reply:
ok , will change.